posthog/types.py

```diff
@@ -0,0 +1,9 @@
+from typing import Union
+
+from posthog.models.filters.filter import Filter
+from posthog.models.filters.path_filter import PathFilter
+from posthog.models.filters.retention_filter import RetentionFilter
+from posthog.models.filters.stickiness_filter import StickinessFilter
+
+Filter_type = Union[Filter, PathFilter, RetentionFilter, StickinessFilter]
```

Comment:
The typing module uses `TitleCase` it seems for types - it might make sense to use it as well.

Reply:
yeah good point!